src/_lib/sass/src/_modals.scss

```diff
@@ -133,3 +124,6 @@ body.modal-open {
         overflow-y: auto;
     }
 }
+.modal-open .modal {
+    backdrop-filter: blur(4px);
```

Comment:
Support seems limited for this, i.e. no Firefox support. I'd suggest looking for alternatives until Colm is back and then discuss the support issue when he's back

Reply:
Yeah, I'm aware, left him a [note](https://newswhip.atlassian.net/browse/RED-691?focusedCommentId=27858).